### PR TITLE
Chap 13

### PR DESCRIPTION
**Pull request by ducbet**

### CODE REVIEW

File: app/views/microposts/_micropost.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <span class="content">
+    <%= micropost.content %>
+    <%= image_tag micropost.picture.url if micropost.picture? %>
+  </span>
+  <span class="timestamp">
+    <%= t "microposts.posted" %> <%= time_ago_in_words(micropost.created_at) %>
+    <%= t "microposts.ago" %>
+    <% if current_user.current_user? micropost.user %>
+      <%= link_to t("microposts.delete"), micropost, method: :delete,
+                                       data: {confirm: t("microposts.sure?")} %>
```
Comment (thaotonto) [inline app/views/microposts/_micropost.html.erb]: indent

File: app/views/static_pages/home.html.erb [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
+    </div>
+  </div>
+<% else %>
+  <div class="center jumbotron">
+    <h1><%= t ".welcome" %></h1>
+    <h2><%= t ".info_html" %></h2>
+    <%= link_to t(".signup_now"), signup_path, class: "btn btn-lg btn-primary" %>
+  </div>
+  <%= link_to image_tag("rails.png", alt: "Rails logo"),
+              "http://rubyonrails.org/" %>
```
Comment (thaotonto) [inline app/views/static_pages/home.html.erb]: same mo same mo

File: app/views/user_mailer/account_activation.html.erb [hunk 3]
```diff
@@ -2,9 +2,8 @@
...
 
-<p>
-  <%= t ".content" %>
-</p>
+<p><%= t ".content" %></p>
 
-<%= link_to t(".activate"), edit_account_activation_url(id: @user.activation_token,
-                                                    email: @user.email) %>
+<%= link_to t(".activate"),
+    edit_account_activation_url(id: @user.activation_token,
```
Comment (thaotonto) [inline app/views/user_mailer/account_activation.html.erb]: here

File: app/views/user_mailer/account_activation.text.erb [hunk 4]
```diff
@@ -2,4 +2,5 @@
 
 <%= t ".intro" %>
 
-<%= edit_account_activation_url(id: @user.activation_token, email: @user.email) %>
+<%= edit_account_activation_url id: @user.activation_token,
+                                email: @user.email) %>
```
Comment (thaotonto) [inline app/views/user_mailer/account_activation.text.erb]: too

File: app/views/user_mailer/password_reset.html.erb [hunk 5]
```diff
@@ -3,7 +3,7 @@
 <p><%= t ".click_link_below" %></p>
 
 <%= link_to t(".reset"), edit_password_reset_url(id: @user.reset_token,
-                                                      email: @user.email) %>
+                                                  email: @user.email) %>
```
Comment (thaotonto) [inline app/views/user_mailer/password_reset.html.erb]: koko

**Comment by ducbet** [comment]
ready

**Comment by ducbet** [comment]
ready

**Pull request merged**